src/module/cp/cf.py

```
@@ -1,22 +1,78 @@
+from src.core.util.tools import check_is_int, get_simple_qrcode, png2jpg, format_int_delta
+from src.data.cf_duel import CFUser, get_binding, establish_binding, accept_binding, settle_duel
+from src.data.model.binding import BindStatus
+from src.platform.online.codeforces import Codeforces, ProbInfo
 
-__cf_version__ = "v3.1.2"
+__cf_version__ = "v5.0.0"
 
 _CF_HELP = '\n'.join(HelpStrList(Constants.help_contents["codeforces"]))
+_CF_DUEL_HELP = '\n'.join([
+    ("/cf duel start [标签|all] (难度) (new): "
+    "开始对战，题目从 Codeforces 上随机选取. 标签中间不能有空格，支持模糊匹配. 难度为整数或一个区间，格式为 xxx-xxx. "
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)